File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C+DS/codes/Recursion/p15.c


#include<stdio.h>
int fun(int n){

	if(n<=1){
		return 1;
	}

return n+fun(n-1)+fun(n-2);
}
void main(){
	int num;
	printf("enter the number \n");
	scanf("%d",&num);

	int ret=fun(num);
	printf("%d\n",ret);
}

File: java-main/java-main/Nestedforloop/p15.java
/*  
  
    10
    9 8
    7 6 5 
    4 3 2 1 
    */

class Demo{

        public static void main(String[]args){

                int N=4;
                int num=10;

                for(int i=1;i<=N;i++){
                        for(int j=1;j<=i;j++){
                                System.out.print(num-- + " ");
                        }
                        System.out.println();
                }
        }
}


